data/darktableconfig.xml.in

```diff
@@ -986,19 +986,6 @@
     <shortdescription>position of the image information line</shortdescription>
     <longdescription/>
   </dtconfig>
-  <dtconfig prefs="darkroom" restart="true">
```

Comment:
You need to keep the entry as it will be used to validate the possible value for it. The only thing to do here is to remove the prefs & restart attributes.

Reply:
even if it's not used anymore (except to retrieve old layout) ?

Reply:
Indeed, it's only for old layout. Forget about that !